### DISCUSSION TITLE
Token request for service user fails without openid in scope

### DISCUSSION THREAD

**Discussion by manga1a** [zitadel/zitadel#8068] [Q&A] [answered]
I have few service users setup to request tokens using only following scope.
`urn:zitadel:iam:org:project:id:{{projectId}}:aud urn:zitadel:iam:org:projects:roles`
This used to work until last week (31st June). Now the request fails with the error: _invalid_scope, Description: The scope openid is missing in your request. Please ensure the scope openid is added to the request. If you have any questions, you may contact the administrator of the application._
It would take much time if I request this change from client side. Is there any configuration I can do in admin panel so the original scope values continue to work?

**Comment by livio-a** [comment]
Hey @manga1a 

If I'm not mistaken, you reached out to our support in the meantime.

I wanted to answer here as well in case somebody else stumbles upon this. As of now there's no configuration option for this.
We will however look into the problem this week and it's likely that we also provide a fix for this within some days.

**Reply by livio-a** [reply]
@muhlemmer ☝️

**Reply by muhlemmer** [reply]
Working on a solution that drops the `openid` scope requirement here: https://github.com/zitadel/oidc/pull/613

**Reply by livio-a** [reply, marked as answer]
We released [v2.54.3](https://github.com/zitadel/zitadel/releases/tag/v2.54.3) and [v2.53.6](https://github.com/zitadel/zitadel/releases/tag/v2.53.6) which allow requests without the `openid` scope again.
